src/dlgConnectionProfiles.cpp

```diff
@@ -640,6 +640,17 @@ QString dlgConnectionProfiles::getDescription(const QString& hostUrl, const quin
         return QLatin1String(
                 "Midnight Sun is a medieval fantasy LPmud that has been around since 1991. We are a non-PK, hack-and-slash game, cooperative rather than competitive in nature, and with a strong "
                 "sense of community.");
+    } else if (hostUrl == QStringLiteral("reinosdeleyenda.es")) {
+        return QLatin1String(
```

Comment:
Would you mind providing a Spanish translation (as a comment) so that I can drop it in as soon as I finally get #1334  in (so I have a test case for the "Connection Preferences" dialog which the GUI language gets changed in the "Profile Preferences" one)... :smiley: